packages/errors/src/errors.ts

```diff
@@ -279,7 +279,23 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/parallel-disallowed`
   },
-  CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH: (arg1: {runUrl: string, parameters: any}) => {
+  CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH: (arg1: {runUrl: string, parameters: any, payload: any }) => {
+    let params: any = arg1.parameters
+
+    if (arg1.payload?.differentParams) {
+      params = {}
+
+      _.map(arg1.parameters, (value, key) => {
+        if (key === 'specs') {
+          params['differentSpecs'] = arg1.payload.differentSpecs ?? []
```

Comment:
why is show this attribute as the negative where as the? Are we able to differentiate if a new spec was sent or a specific spec was missing from the payload?

Reply:
We're able to get the different values in the cloud, keeping track of the specs sent by the TR

Reply:
so diff specs is only tied to the current run that errored right?

How does this look if it sent 1 additional spec vs sent 1 spec less?

Reply:
When there's one spec extra, its gonna show that spec extra, and the same when there's one less, it shows the spec missing